bin/test/main.ml

```diff
@@ -118,7 +131,7 @@ let run_cram_tests ?syntax t ?root ppf temp_file pad tests =
       let n = run_test ?root blacklist temp_file test in
       let lines = read_lines temp_file in
       let output =
-        let output = List.map (fun x -> `Output x) lines in
+        let output = List.map (fun x -> `Output (trim_end x)) lines in
```

Comment:
Might be worth extracting a `output_from_lines` function so we don't have to remember to update all those call sites in the future!